yb-voyager/src/query/queryissue/parser_issue_detector.go

```diff
@@ -472,6 +473,55 @@ func (p *ParserIssueDetector) genericIssues(query string) ([]QueryIssue, error)
 	return result, nil
 }
 
+func (p *ParserIssueDetector) GetRedundantIndexIssues(redundantIndexes []utils.RedundantIndexesInfo) []QueryIssue {
+
+	redundantIndexToInfo := make(map[string]utils.RedundantIndexesInfo)
```

Comment:
nit: Since we're already inside the `GetRedundantIndexIssues` function, you can skip the "redundant" prefix to variable names inside this func. 
They seem borderline long, and is slightly hard to read. 
(haven't thought about this too much though, you might need to distinguish between "redundant" and existing" in certain situations"?)

Reply:
> (haven't thought about this too much though, you might need to distinguish between "redundant" and existing" in certain situations"?)

yes it is required to differentiate between redundant and existing index, but let me see if I can clear it up any further.